cassandra-protocol/src/types/value.rs

```diff
@@ -360,6 +359,29 @@ where
     }
 }
 
+impl<V> From<BTreeSet<V>> for Bytes
```

Comment:
Also - we need a version for HashSet.

Reply:
Regarding my comment above about representing sets as a vector again: We can keep this method though, so it's possible to convert `HashSet`s and `BTreeSet`s into our cdrs-tokio set type.

Reply:
Ok, so then we're back to the original question - how such Set is different from a List?

Reply:
Because it will be helpful to know explicitly whether something is a `Set` or `List`, even if they behave the same.

Reply:
Here's an example of where this caused us a problem, this unit test expects a list of sets but because set's are represented by the list type in cdrs-tokio, it fails.

![image](https://user-images.githubusercontent.com/39339036/186582503-c2da9d62-11f3-4548-ba81-7f8f3d04b4ad.png)